miniupnpc/miniupnpc.c

```diff
@@ -706,13 +706,13 @@ UPNP_GetIGDFromUrl(const char * rootdescurl,
 	int descXMLsize = 0;
 
 	descXML = miniwget_getaddr(rootdescurl, &descXMLsize,
-	                           lanaddr, lanaddrlen, 0, NULL);
+	   	                       lanaddr, lanaddrlen, 0, NULL);
```

Comment:
This violates the coding standard dictated by surrounding code. It is also not a semantic change.